dockerHashing/src/main/scala/cromwell/docker/DockerInfoActor.scala

```diff
@@ -145,7 +146,7 @@ final class DockerInfoActor(
     Queue.boundedNoneTerminated[IO, DockerInfoContext](queueBufferSize) map { queue =>
       val source = queue.dequeue
       // If the registry imposes throttling, debounce the stream to ensure the throttling rate is respected
-      val throttledSource = registry.config.throttle.map(_.delay).map(source.debounce[IO]).getOrElse(source)
+      val throttledSource = registry.config.throttle.map(_.delay).map(source.metered[IO]).getOrElse(source)
```

Comment:
Per the scaladoc:

```
Throttles the stream to the specified `rate`. Unlike [[debounce]], [[metered]] doesn't drop elements.
```

Not dropping elements (Docker image hash requests) is highly desirable.

Reply:
Good catch!